app/src/main/java/xyz/jhughes/laundry/LaundryParser/Rooms.java

```diff
@@ -28,10 +28,10 @@ private Rooms() {
             roomsToImage.put(room, toImageResourceId(room));
         }
 
-        machineAvailabilityColors.put("Available", R.color.Available);
-        machineAvailabilityColors.put("In use", R.color.InUse);
-        machineAvailabilityColors.put("Almost done", R.color.AlmostDone);
-        machineAvailabilityColors.put("End of cycle", R.color.Finished);
+        machineAvailabilityColors.put(MachineStates.AVAILABLE, R.color.Available);
```

Comment:
Yeah, But again the lack of a context in all relevant places makes it rough. We may need to init these in Activity.onCreate() and then access statically like this. But even that isnt COMPLETELY ideal.